src/content/structured/get-started/react.mdx

```diff
@@ -103,3 +103,38 @@ module.exports = withTM(nextConfig);
 The `@ukic/react` package will need to be transformed before you can use these components in Jest tests.
 
 Add a `transformIgnorePatterns` field with the value `["/node_modules/(?!@ukic/react)"]` to your Jest config.
+
+## Working with slotted SVGs
+
+When wanting to slot an SVG into one of the components, you should import SlottedSVG in your React files to prevent an error stating that property 'slot' doesn't exist.
```

Comment:
```suggestion
To slot an SVG into one of our React components, import the `SlottedSVG` component from `@ukic/react` into your React files. This will prevent an error stating that property 'slot' doesn't exist.
```